### PR TITLE
support tuple/list init for VarBase

### PR DESCRIPTION
**Pull request by zhwesky2010**
### PR types

Function optimization
### PR changes

APIs
### Describe

Varbase supports list/tuple initialization.

### CODE REVIEW

File: python/paddle/fluid/dygraph/base.py [hunk 1]
```diff
@@ -530,19 +531,19 @@ def check_in_out(in_out_list, name):
 
 
 @framework.dygraph_only
-def to_variable(value, name=None, zero_copy=None):
+def to_variable(value, name=None, zero_copy=None, dtype=None):
```
Comment (zhiqiu) [inline python/paddle/fluid/dygraph/base.py:534]: Add `dtype` in comments.

Comment (zhwesky2010) [reply python/paddle/fluid/dygraph/base.py:534]: Done, thx

**Review by zhiqiu — APPROVED** [review]
LGTM

**Review by lanxianghit — APPROVED** [review]

**Review by saxon-zh — APPROVED** [review]
LGTM

**Pull request merged**